src/lib/features/frontend-api/frontend-api-service.ts

```diff
@@ -208,6 +208,23 @@ export class FrontendApiService {
         );
     }
 
+    async setFrontendCorsSettings(
+        value: FrontendSettings['frontendApiOrigins'],
+        auditUser: IAuditUser,
+    ): Promise<void> {
+        const error = validateOrigins(value);
+        if (error) {
+            throw new BadDataError(error);
+        }
+        const settings = (await this.getFrontendSettings()) || {};
```

Comment:
I noticed get frontend settings uses a cached value by default so either:
```suggestion
        const settings = (await this.getFrontendSettings(false)) || {};
```

or 


```suggestion
        const settings = (await this.fetchFrontendSettings()) || {};
```